demo/api/src/warnings/entities/warning.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { RootBlockEntity } from "@comet/blocks-api";
+import { CrudField, CrudGenerator } from "@comet/cms-api";
+import { BaseEntity, Entity, Enum, OptionalProps, PrimaryKey, Property } from "@mikro-orm/core";
+import { Field, ID, ObjectType } from "@nestjs/graphql";
+import { v4 as uuid } from "uuid";
+
+import { WarningLevel } from "./warning-level.enum";
+import { WarningState } from "./warning-state.enum";
+
+@ObjectType()
+@Entity()
+@RootBlockEntity<Warning>()
+@CrudGenerator({ targetDirectory: `${__dirname}/../generated/`, requiredPermission: ["pageTree"] })
+export class Warning extends BaseEntity<Warning, "id"> {
+    [OptionalProps]?: "createdAt" | "updatedAt" | "status";
+
+    @PrimaryKey({ type: "uuid" })
+    @Field(() => ID)
+    id: string = uuid();
+
+    @Property()
+    @Field()
+    createdAt: Date = new Date();
+
+    @Property({ onUpdate: () => new Date() })
+    @Field()
+    updatedAt: Date = new Date();
+
+    @Property()
+    @Field()
+    @CrudField()
+    type: string;
```

Comment:
I could think of:
 - Block
 - Job
 - Performance Issue
 - Types defined in the Project, maybe rate limit exceeded for a specific API.
 

I'm not quite sure whether the type makes sense like that when everything is finished. I first had it as an enum but changed it to string then. Maybe it will have to be adjusted later. Unless you have any suggestions for improvement now ?